packages/menu/src/MenuList/MenuList.js

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import { themeGet } from 'styled-system'
+import { Flex, applySizes } from 'pcln-design-system'
+
+const sizes = {
+  singleColumn: css`
+    & > * {
+      width: 100%;
+    }
+  `,
+  twoColumns: css`
+    & > * {
+      width: calc(50% - ${themeGet('space.1')}px);
+    }
+  `,
+}
+
+const MenuContainer = styled(Flex)`
+  font-family: 'Montserrat', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
is this necessary?

Reply:
Yeah the popover content gets confused, we should create a ticket to look into this

Reply:
Not sure if it's strictly necessary, but fwiw, `popover` renders its content in a React portal, so that might have something to do with it.

Reply:
I thought that had been fixed at some point 🤔